sw/device/lib/testing/usb_testutils_streams.c

```diff
@@ -242,19 +236,9 @@ static void strm_rx(void *stream_v, dif_usbdev_rx_packet_info_t packet_info,
       switch (read_method) {
 #if USBUTILS_MEM_FASTER
         // Faster read performance using custom routine
-        case kReadMethodFaster: {
-          const uint32_t USBDEV_BUFFER_REG_OFFSET = 0x800U;
-          uint32_t offset =
-              USBDEV_BUFFER_REG_OFFSET + (buf.id * USBDEV_MAX_PACKET_SIZE);
-          usbutils_memcpy_from_mmio32(usbdev->dev->base_addr, offset, data,
-                                      len);
-          buf.offset += len;
-          buf.remaining_bytes -= len;
-          bytes_read = len;
-          // We must still return the buffer to the pool
-          CHECK_DIF_OK(
-              dif_usbdev_buffer_return(usbdev->dev, usbdev->buffer_pool, &buf));
-        } break;
+        case kReadMethodFaster:
+          // TODO: faster read method not yet integrated, defaulting to standard
+          // no break
```

Comment:
Would you mind squashing this change into the first commit into this PR?

Reply:
Yah, the later commits change the previous one, so I also think they should be squashed.